.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,25 +11,24 @@ body:
       - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported. If you don't find a relevant match or if you're unsure, don't hesitate to **open a new issue**. The bugsquad will handle it from there if it's a duplicate.
       - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
 
-- type: input
-  attributes:
-    label: Godot version
-    description: >
-      Specify the Git commit hash if using a development or non-official build.
-      If you use a custom build, please test if your issue is reproducible in official builds too.
-    placeholder: 3.3.stable, 4.0.dev (3041becc6)
-  validations:
-    required: true
-
-- type: input
+- type: textarea
   attributes:
-    label: System information
+    label: Environment information
```

Comment:
And so this section should probably be called "System information" as before, and references to the Godot version can be dropped from it.

Reply:
If I extract godot version again (which is probably better), then that rename should happen too, yes.

Reply:
Done